Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml

```diff
@@ -2499,36 +2638,80 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "69":
-    id: "69"
-    taskid: 44e0b4cc-b79d-4a42-8f59-30796d55810e
-    type: condition
+  "73":
+    id: "73"
+    taskid: 511d52a4-f1f3-4164-8cb7-fdbea90dd7ef
+    type: playbook
     task:
-      id: 44e0b4cc-b79d-4a42-8f59-30796d55810e
+      id: 511d52a4-f1f3-4164-8cb7-fdbea90dd7ef
       version: -1
-      name: Check if Cortex XDR - IR is Enabled
-      description: Returns 'yes' if integration brand is available. Otherwise returns
-        'no'
-      scriptName: IsIntegrationAvailable
-      type: condition
+      name: Cortex XDR - Execute commands
```

Comment:
Need to change the name of the task of the subplaybook

Reply:
fixed